codacy/values-microk8s.yaml

```diff
@@ -33,9 +33,12 @@ listener:
   resources:
     limits:
       cpu: 500m
+  # These configurations are only used if the `nfsserverprovisioner` is enabled
   persistence:
     claim:
-      size: 50Gi
+      # This is the size of the claim that the component will get from the provisioner (if used)
+      # Therefore this value needs to be smaller than `nfsserverprovisioner.persistence.size`
+      size: 90Gi
```

Comment:
do we have those values summed up in the hw requirement documentation?

Reply:
we are just asking for the 200Gi